examples/pennylane/7_Pulse_control_OQC_Lucy/7_Pulse_control_OQC_Lucy.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Since this runs on hardware let's add a cost tracker section. See an example in the first and last cells here: <a href="https://github.com/amazon-braket/amazon-braket-examples/blob/main/examples/pennylane/1_Parallelized_optimization_of_quantum_circuits/1_Parallelized_optimization_of_quantum_circuits.ipynb" target="_blank">https://github.com/amazon-braket/amazon-braket-examples/blob/main/examples/pennylane/1_Parallelized_optimization_of_quantum_circuits/1_Parallelized_optimization_of_quantum_circuits.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
great idea!